dcplib/aws/sqs_handler.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+import boto3
+
+
+class SQSHandler:
+    """
+    A class encapsulating the behaviors associated with interacting with an SQS Queue object and raising errors when
+    queue-related behaviors fail.
+    """
+
+    def __init__(self, queue_url):
+        self.sqs = boto3.resource('sqs')
+        self.queue_url = queue_url
+
+    def add_message_to_queue(self, payload):
+        """ Given a payload, add it to the queue. Does not return anything, but will raise an exception if the
+        addition was not successful."""
+
+        response = self.sqs.meta.client.send_message(QueueUrl=self.queue_url,
```

Comment:
Why not use the resource API and the lazy caching loaders in this directory? Partial sketch:
```python
from . import resources, clients

def __init__(self, queue_name=None, queue_url=None):
    if queue_name:
        self.queue = resources.sqs.Queue(clients.sqs.get_queue_url(QueueName=queue_name))
    else:
        self.queue = resources.sqs.Queue(queue_url)

def send(self, msg, **attributes):
    res = self.queue.send_message(MessageBody=json.dumps(msg), MessageAttributes={k: {"StringValue": v} for k, v in attributes.items()})
    return res['MessageId']
```

Reply:
Done.